jenkins-pipelines/rolling-upgrade-with-null-inserts.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+rollingUpgradePipeline(
+    backend: 'gce',
+    base_versions: ['4.1'],
+
+    test_name: 'upgrade_test.UpgradeTest.test_generic_cluster_upgrade',
+    test_config: '''["test-cases/upgrades/generic-rolling-upgrade.yaml", "test-cases/upgrades/rolling-upgrade-with-null-inserts.yaml"]''',
```

Comment:
For some reason I have an issue here with the export, although I copied it from a similar test that using it (EaR longevity).

Reply:
export 'SCT_CONFIG_FILES=[test-cases/upgrades/generic-rolling-upgrade.yaml,' 'configurations/rolling-upgrade-with-null-inserts.yaml]'
03:15:44  + SCT_CONFIG_FILES='[test-cases/upgrades/generic-rolling-upgrade.yaml,'
03:15:44  /jenkins/slave/workspace/scylla-master/reproducers/rolling-upgrade-with-null-inserts/scylla-cluster-tests@tmp/durable-32eee7f7/script.sh: line 10: export: `configurations/rolling-upgrade-with-null-inserts.yaml]': not a valid identifier

Reply:
@fruch any idea why?

Reply:
can you share the link ?

Reply:
it's a bug in the `rollingUpgradePipeline.groovy` 

in other pipelines the test config is handled correctly:
```groovy
test_config = groovy.json.JsonOutput.toJson(params.test_config)
```